.github/workflows/edr-benchmark.yml

```diff
@@ -35,7 +35,8 @@ jobs:
     environment: github-action-benchmark
     runs-on: self-hosted
     # Only run for trusted collaborators since third-parties could run malicious code on the self-hosted benchmark runner.
-    if: github.ref == 'refs/heads/main' || (github.event.pull_request.author_association == 'OWNER' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'COLLABORATOR')
+    # if: github.ref == 'refs/heads/main' || (github.event.pull_request.author_association == 'OWNER' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'COLLABORATOR')
```

Comment:
Why was this commented?

Reply:
Sorry, I thought I had added a comment about this but clearly I didn't. I think this job is not working with the new repo; my guess is that some extra configuration is needed. I opened https://github.com/NomicFoundation/edr/issues/402 about it.